### PR TITLE
Update UserNavigationListener.php

### PR DESCRIPTION
**Pull request by pressi**
Change the Security Class use.
Install Error in Contao Version 5.4.0-RC1

### CODE REVIEW

**Review by fritzmg — CHANGES_REQUESTED** [review]
This makes it incompatible with Contao 4.13.

You should instead change

```php
private readonly Security $security,
```

to

```php
private readonly AuthorizationCheckerInterface $security,
```

(+ the use statement). This will be compatible with all Symfony versions.

**Comment by pressi** [comment]
ok, good to know, thanks @fritzmg.
i have change this.

**Review by fritzmg — APPROVED** [review]

File: src/EventListener/UserNavigationListener.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ public function __construct(
         private readonly Connection $connection,
         private readonly RequestStack $requestStack,
         private readonly ScopeMatcher $scopeMatcher,
-        private readonly Security $security,
+        private readonly AuthorizationCheckerInterface $security,
```
Comment (aschempp) [inline src/EventListener/UserNavigationListener.php:29]: Please also change the variable name and its usage in the code.

```suggestion
        private readonly AuthorizationCheckerInterface $authorizationChecker,
```

**Review by aschempp — CHANGES_REQUESTED** [review]

**Review by aschempp — APPROVED** [review]

**Comment by aschempp** [comment]
Thank you @pressi è

**Pull request merged**